### ISSUE TITLE
Add mouse holding support

### ISSUE THREAD

**Issue by yairEO** [picodealion/game-of-life#1]
so when I hold the mouse it colors/uncolors a square, so I won't have to click click click at each tiny one.

on `mousedown` event bind a `mouseenter` event and listen. on `mouseup` event remove the `mouseenter` event

**Comment by stephanmullerNL** [comment]
Good idea! I still have a lot that I want to add, but I think this one shouldn't be too hard
